extension/scripts/global/functions/formatting.js

```diff
@@ -258,11 +258,11 @@ function formatNumber(number, options = {}) {
 
 	if (!text) text = abstract.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
 
-	// if (options.currency) {
-	// 	return `${number >= 0 ? (options.forceOperation ? "+" : "") : "-"}$${Math.abs(number)
-	// 		.toString()
-	// 		.replace(/\B(?=(\d{3})+(?!\d))/g, ",")}`;
-	// }
+	if (options.currency) {

```

Comment:
This code is obsolete. Both the operation and the currency are handled in other parts of the code.